stories/ImagesList/ImagesList.stories.tsx

```diff
@@ -0,0 +1,666 @@
+import React from 'react';
+import { Icon } from '../../src';
+
+export const ElementoSingolo = () => (
+  <div className='row'>
+    <div className='col-12 col-lg-4'>
```

Comment:
I would use the `reactstrap` components here rather than simple html markup:
```suggestion
  <Row>
    <Col lg={4} ...>
```

And so on for similar markup

Reply:
Fixed